src/main/java/com/odysseusinc/arachne/datanode/config/HibernateValidatorJpaConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ * Copyright 2019 Odysseus Data Services, inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin, Vitaly Koulakov, Anastasiia Klochkova, Sergej Suvorov, Anton Stepanov
+ * Created: Oct 16, 2019
+ *
+ */
+
+package com.odysseusinc.arachne.datanode.config;
+
+import java.util.Map;
+import java.util.Objects;
+import javax.sql.DataSource;
+import javax.validation.ValidatorFactory;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.JpaProperties;
+import org.springframework.boot.autoconfigure.transaction.TransactionManagerCustomizers;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.transaction.jta.JtaTransactionManager;
+
```

Comment:
please add Java doc, it is really unclear why do we need this config

Reply:
Not sure javadoc is suitable here, could be just a comment is more convenient?

Reply:
I don't really mind, but why java doc is not suitable?

Reply:
I reckon nobody would generate javadoc and read it then. 
And one more cent, where should be javadoc tag placed? 
When making a class description then it would be not obvious where the magic does.
Placing as method description also might disrupt clearance since it's not a method description.
So I've added as comment close to the hibernate parameter injection.